doc/nrf/known_issues.rst

```diff
@@ -326,6 +326,22 @@ Bluetooth LE
 
 .. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
+NCSDK-8224: Security changed and pairing failed callbacks are not always called
+  The pairing failed and security changed callbacks are not called when disconnected during the pairing procedure or required security was not met.
```

Comment:
When what is disconnected?

Reply:
during the pairing procedure

Reply:
Right, but **what** is disconnected?
"when the application is disconnected"? "when the connection between the devices is interrupted"?

Reply:
```suggestion
  The pairing failed and security changed callbacks are not called when the connection has been disconnected during the pairing procedure or required security was not met.
```

Sorry, I misread your question. The connection is disconnected. :) 
The above suggestion matches the language for the disconnected callback:
 `/** @brief A connection has been disconnected.`

Reply:
OK, thanks. :)
Let's go for this then:
```suggestion
  The pairing failed and security changed callbacks are not called when the connection is disconnected during the pairing procedure or the required security is not met.
```